resources/js/Pages/Home/Index.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+    <Nav :show="true" />
+    <div v-if="events.data.length" class="overflow-hidden sm:rounded-md">
+        <ul>
+            <Event
```

Comment:
I think we should just link to the event external URL here instead of an event details page.